### PR TITLE
[DOCs/operators]: Release notes v2024.14-crunch & config score updates

### PR DESCRIPTION
**Pull request by serinko**
documenting changes with `crunch` release

This change is [Reviewable](https://reviewable.io/reviews/nymtech/nym/5222)

### CODE REVIEW

File: documentation/docs/pages/operators/changelog.mdx [hunk 1]
```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
...
 
+## `v2024.14-crunch`
+
+- [Release binaries](https://github.com/nymtech/nym/releases/tag/nym-binaries-v2024.14-crunch)
+- [`nym-node`](nodes/nym-node.mdx) version `1.2.0`
+- [Releae operators updates and tools](changelog#operators-updates--tools)
+- [Github CHANGELOG.md](https://github.com/nymtech/nym/blob/nym-binaries-v2024.14-crunch/CHANGELOG.md)
+
+```sh
+
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/changelog.mdx]: Presuming left deliberately, still flagging to double check

File: documentation/docs/pages/operators/changelog.mdx [hunk 2]
```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
...
+| [clap_complete](https://github.com/clap-rs/clap) | `4.5.29` | `4.5.33` | 
+| [pin-project](https://github.com/taiki-e/pin-project) | `1.1.5` | `1.1.6` | 
+| [serde](https://github.com/serde-rs/serde) | `1.0.210` | `1.0.211` | 
+| [serde_json](https://github.com/serde-rs/json) | `1.0.128` | `1.0.132` | 
+| [wasm-bindgen](https://github.com/rustwasm/wasm-bindgen) | `0.2.93` | `0.2.95` | 
+| [wasm-bindgen-futures](https://github.com/rustwasm/wasm-bindgen) | `0.4.43` | `0.4.45` | 
+| [web-sys](https://github.com/rustwasm/wasm-bindgen) | `0.3.70` | `0.3.72` |   
+| Updates `anyhow` | `1.0.89` | `1.0.90` |
+
+[[Product Data] Introduce data persistence on gateways](https://github.com/nymtech/nym/pull/5022): This PR builds on top of [\#4974](https://github.com/nymtech/nym/pull/4974), not changing the behavior of the data collection, but persisting them in a sqlite database so they can be kept across restarts and crashes. It also leave the door open for other stats module to use that storage if needed. Here are some points of interest:
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/changelog.mdx]: Is this missing a `-`?

File: documentation/docs/pages/operators/changelog.mdx [hunk 3]
```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
...
+
+- [chore: ecash contract migration to remove unused 'redemption_gateway_share'](https://github.com/nymtech/nym/pull/5104)
+
+- [[Product Data] Client-side stats collection ](https://github.com/nymtech/nym/pull/5107): The goal is to anonymously gather stats from nym-clients. These stats will be sent through the mixnet to a Nym run service provider that will gather them. This PR sets the scene to send stats in a mixnet message to an address. The address can be set when the client is created. Current stats include some infos on sent packets along with platform information. If a receiving address is set, the client will send a mixnet packet every 5min to this address. Otherwise, nothing happens and the client runs as usual. 
+
+- [Send mixnet packet stats using task client](https://github.com/nymtech/nym/pull/5109)
+ 
+- [Add granular log on nym-node](https://github.com/nymtech/nym/pull/5111) and make use of it for `defguard_wireguard_rs` big info logs
+
+[Rewarding for ticketbook issuance](https://github.com/nymtech/nym/pull/5112): Revamps the current validator rewarder to allow for rewards for issuing the zk-nym ticketbooks.
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/changelog.mdx]: Is this missing a `-`?

File: documentation/docs/pages/operators/changelog.mdx [hunk 4]
```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
...
+Then run the jokes in a new window for control
+```sh
+$HOME/nym-binaries/network_tunnel_manager.sh joke_through_the_mixnet
+$HOME/nym-binaries/network_tunnel_manager.sh joke_through_wg_tunnel
+```
+</AccordionTemplate>
+
+#### Tools
+
+- **[New APIs documentation](../apis/introduction)** with interactive APIs generated from the OpenAPI specs of various API endpoints offered by bits of Nym infrastructure run both by Nym and community operators for both Mainnet and the Sandbox testnet.
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/changelog.mdx:302]: we're linking to a link that doesn't exist yet in this branch, but if we merge fast I suppose that's fine, vs changing the target of the OpenAPI doc PR ?

File: documentation/docs/pages/operators/changelog.mdx [hunk 5]
```diff
@@ -25,14 +25,276 @@ export const CiConfig = () => (
...
+- [Nym Harbourmaster](https://harbourmaster.nymtech.net/) has a new tab called `CONTRACT EXPLORER` querying data from Nym mixnet contract in real time.
+- [Nym Explorer](https://explorer.nymtech.net) is updated to read migrated nodes correctly
+- [New community explorer by SpectreDAO](https://explorer.nym.spectredao.net/dashboard) offers Nym Network dashboard, Node overview and Account stats view functions for operators and delegators.
+- [`nym-vpnc`](../developers/nymvpncli) build and run documentation, for those who don't want to use the Nym VPN GUIs.
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/changelog.mdx:306]: ditto

File: documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx [hunk 6]
```diff
@@ -126,31 +131,91 @@ This is a quick summary, to understand the full picture, please see detailed [*R
...
+### Rewarded Set Selection
 
 For a node to be rewarded, the node must be part of a [Rewarded set](https://validator.nymtech.net/api/v1/epoch/reward_params) (which currently = active set) in the first place. The active set is selected in the beginning of each epoch (every 60min) where total of 240 Nym nodes - represented by 120 mixnodes and 120 gateways, are randomly allocated across the layers.
 
-The algorithm choosing nodes into the active set takes into account node's performance and [stake saturation](../tolkenomics.mdx#stake-saturation), both values being between 0 and 1 and config score which is either 0 or 1.
+The algorithm choosing nodes into the active set takes into account these parameters:
+
+1. [Config score](#config-score-calculation)
+2. [Performance](#performance-calculation)
+3. [Stake saturation](../tolkenomics.mdx#stake-saturation)
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx]: typo in 'tolkenomics'

File: documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx [hunk 7]
```diff
@@ -126,31 +131,91 @@ This is a quick summary, to understand the full picture, please see detailed [*R
...
+> **active_set_selection_probability = config_score \* ( node_performance ^ 20 ) \* stake_saturation**
+</Callout>
+
+#### Config Score Calculation
+
+The nodes selection to the active set has a new parameter - `config_score`. Config score currently looks into three paramteres:
 
-**Config score is introduced:** The nodes selection to the active set has a new parameter - `config_score`. Config score currently looks if the node binary is `nym-node` (not legacy `nym-mixnode` or `nym-gateway`) **AND** if [Terms & Conditions](nodes/nym-node/setup.mdx#terms--conditions) are accepted. Config score has binary values of either 0 or 1, with a following logic:
+1. If the node binary is `nym-node` (not legacy `nym-mixnode` or `nym-gateway`)
+2. If [Terms & Conditions](nodes/nym-node/setup.mdx#terms--conditions) are accepted.
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx]: 404

File: documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx [hunk 8]
```diff
@@ -126,31 +131,91 @@ This is a quick summary, to understand the full picture, please see detailed [*R
...
+> **0.995 ^ ( ( X * versions_behind ) ^ 1.65 )**
+>
+> where: <br />
+> **X = 1;       for patches** <br />
+> **X = 10;      for minor versions** <br />
+> **X = 100;     for major versions**
 </Callout>
 
-For a comparison we made an example with 5 nodes, where first number is node performance and second stake saturation (assuming all of them `config_score` = 1 and not 0):
+> The exact parameters are live accessible on `/v1/status/config-score-details`
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx]: Could make this a link to https://validator.nymtech.net/api/swagger/index.html#/Status/config_score_details

File: documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx [hunk 9]
```diff
@@ -126,31 +131,91 @@ This is a quick summary, to understand the full picture, please see detailed [*R
...
+![](/images/operators/tokenomics/reward_version_graph.png)
+
+As you can see on above, the algorithm is designed to give maximum probability (`1`) to the latest version and exponentialy dicrease the probability to non-upgraded nodes where the more important version the node is behind, the faster the cliff. This eliminates any older nodes despite their saturation and performance to take place in the Rewarded set and gives a priority to the operators running up-to-date nodes, ensuring as strong network as possible.
+
+
+#### Performance Calculation
+
+Performance is measured by Nym Network Monitor which sends thousands of packages through different routes every 15 minutes and measures how many were dropped on the way. Test result represents percentage of packets succesfully returned (can be anything between 0 and 1). Performance value is nodes average of these tests in last 24h.
+
+Good performance is much more essential than [stake saturation](../tolkenomics.mdx#stake-saturation), because it's lifted to 20th power in the selection formula.
```
Comment (mfahampshire) [inline documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx]: Ditto typo in `tolkenomics`

**Review by mfahampshire — APPROVED** [review]
minor fixes required otherwise :ship:

File: documentation/docs/pages/operators/changelog.mdx (hunk 4 above)
Comment (serinko) [reply documentation/docs/pages/operators/changelog.mdx:302]: I am aware, I won't deploy it up until all s merged together

File: documentation/docs/pages/operators/changelog.mdx (hunk 5 above)
Comment (serinko) [reply documentation/docs/pages/operators/changelog.mdx:306]: I am aware, I won't deploy it up until all s merged together

**Pull request merged**